src/SimpleJavaApp/MainApp.java

```diff
@@ -1,32 +1,70 @@
 package SimpleJavaApp;
 
 import javax.swing.*;
+import java.awt.*;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.awt.event.FocusAdapter;
+import java.awt.event.FocusEvent;
 
 
-
-public class MainApp {
+public class MainApp extends JFrame {
     private JButton Login;
     private JPanel panelMain;
-    private JPasswordField passwordField1;
-    private JTextField textField1;
+    private JPasswordField password;
+    private JTextField username;
     private JButton loginButton;
     private JButton exitButton;
 
 
     public MainApp() {
+        add(panelMain);
+        setTitle("Please Log in");
+        setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
+        setSize(300, 200);
+
+        this.setLocationRelativeTo(null);
+
         loginButton.addActionListener(new ActionListener() {
             @Override
             public void actionPerformed(ActionEvent e) {
+                String user, pass;
+                user = username.getText();
+                pass = password.getText();
 
+                if (user.equals("admin") && pass.equals("123")) {
```

Comment:
You should create a class to save constants. Ex: AccountUtil.java and call AccountUti.USER_NAME
`public class AccountUtil {
    public static final String USER_NAME = "admin";
}`

Reply:
You should read `naming convention` https://howtodoinjava.com/java/basics/java-naming-conventions/